web_tree_image/static/src/js/web_tree_image.js

```diff
@@ -19,7 +19,7 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-odoo.define('web.tree_image', function (require) {
+odoo.define('web.WebTreeImage', function (require) {
```

Comment:
This only impacts in possible module that inherits this, so no actual problem I think.